python/cudf/cudf/core/column/string.py

```diff
@@ -2236,7 +2236,7 @@ def get(self, i: int = 0) -> SeriesOrIndex:
 
         return self._return_or_inplace(libstrings.get(self._column, i))
 
-    def get_json_object(self, json_path):
+    def get_json_object(self, json_path, allow_single_quotes=False, strip_quotes_from_single_strings=True, missing_fields_as_nulls=False):
```

Comment:
We should require keyword arguments for all of these options.
```suggestion
    def get_json_object(self, json_path, *, allow_single_quotes=False, strip_quotes_from_single_strings=True, missing_fields_as_nulls=False):
```